mindone/models/t5/modules/pretrained_config.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2023 Huawei Technologies Co., Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+"""
+Pretrained config.
+"""
+
+import copy
+import json
+import os
+from typing import Dict, Optional, Tuple
+
+from mindspore import log as logger
+
+from .configs import HF_CONFIG_URL_BASE
+from .download import cached_path
+
+
+class PreTrainedConfig:
```

Comment:
`PreTrainedConfig` here and in `mindone/models/t5/modules/configs.py` are identical. Why is there a need for repetition?